src/index.js

```diff
@@ -28,4 +28,9 @@ const RootedApp = root(tree, App);
 window.tree=tree;
 window.events=events;
 
-ReactDOM.render(<RootedApp />, document.querySelector('#app'));
+superagent.get('/api/user').then((res) => {
+  const user = JSON.parse(res.text);
+  window.STATE_FROM_SERVER = {user};
+
+  ReactDOM.render(<RootedApp />, document.querySelector('#app'));
+})
```

Comment:
this should only be for the development version. Drone loads this automatically inside the index.html file on page load